core/src/commonTest/kotlin/performance/KordEventDropTest.kt

```diff
@@ -63,6 +64,8 @@ class KordEventDropTest {
     )
 
     @Test
+    // This test seems to timeout sometimes on native
+    @IgnoreOnNative
```

Comment:
can we increase/remove the timeout as with js? https://github.com/kordlib/kord/pull/926